Makefile

```diff
@@ -10,10 +10,14 @@ all: build
 
 $(BIN_PATH): FORCE
 	@echo "--> Building Humio CLI"
-	go build -o $(BIN_PATH) cmd/humioctl/*.go
+	cd cmd/humioctl && go build -o $(BIN_PATH)
 
 build: $(BIN_PATH)
 
+build-windows:
+	@echo "--> Building Humio CLI (Windows)"
+	cd cmd/humioctl && GOOS=windows GOARCH=amd64 go build -o $(BIN_PATH).exe
```

Comment:
```suggestion
	GOOS=windows GOARCH=amd64 go build -o $(BIN_PATH).exe ./cmd/humioctl
```

Reply:
That would not work.

Reply:
Why not?
Works on my machine.

Reply:
You are right! The issue I had was that I excluded "./" at on the path. Apparently go build interprets this as a package name and you get `package cmd/humioctl is not in GOROOT (/usr/lib/go-1.13/src/cmd/humioctl)`

Reply:
Yes. The './' is important here.